osrf/vrx
# [Garden] Wildlife scoring plugin - Part2

This pull request ports the WildlifeScoringPlugin from Gazebo classic.

### How to test it?

* Launch the wildlife_task world:
```
ros2 launch vrx_gz competition.launch.py world:=wildlife_task
```

* On a new terminal launch the joystick teleop node:
```
ros2 launch vrx_gz usv_joy_teleop.py
```

* On a new terminal check the `/vrx/task/info` topic:
```
ros2 topic  echo /vrx/task/info
```

Try circumnavigating the buoys (platypus clockwise and turtle counter clockwise). You should see messages of this type:
```
[Dbg] [WildlifeScoringPlugin.cc:409] platypus::link Virtual gate crossed clockwise! (25% completed)
```

**Bonus**: I enabled contact checking again. For some reason the topic subscription was removed.
